content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Cloudify HA Day-2 Operations
+description: Guides for manging Cloudify HA clusters
+weight: 80
+alwaysopen: false
+---
+
+# Day-2 operations for Cloudify HA
+
+## Introduction
```

Comment:
No point in this bullet if we dont have anything under it...
I suggest removing it.